pyintervalxt/test/interval_exchange_transformation.py

```diff
@@ -56,4 +56,36 @@ def test_reduce():
     assert repr(iet2) == "[c: 23] [d: 11] [e: 21] / [e] [d] [c]"
     assert repr(iet3) == "[f: 9] [g: 65] / [g] [f]"
 
+def iet_check(iet):
+    saf = iet.safInvariant()
+    decomposition = intervalxt.DynamicalDecomposition(iet)
+    decomposition.decompose()
+    ncyls = sum(bool(component.cylinder() == True) for component in decomposition.components())
```

Comment:
Did you mean to check something about `ncyls`?

Reply:
Not really. Just that it works

Reply:
I added a bit more.